spec/GameScorerSpec.js

```diff
@@ -0,0 +1,138 @@
+  return the correct total", function() {
+    it("test 1", function() {
+      frameScorer.inputBowls(10,0);
+      game.updateStrike(frameScorer)
+      gameScorer.calculateFirstBowlScore(game, frameScorer)
+      gameScorer.calculateSecondBowlScore(game, frameScorer)
+      game.updateStrikeAndSpareVariables(frameScorer)
+      frameScorer.inputBowls(9,1);
+      game.updateStrike(frameScorer)
+      gameScorer.calculateFirstBowlScore(game, frameScorer)
+      gameScorer.calculateSecondBowlScore(game, frameScorer)
+      game.updateStrikeAndSpareVariables(frameScorer)
+      frameScorer.inputBowls(5,5);
+      game.updateStrike(frameScorer)
+      gameScorer.calculateFirstBowlScore(game, frameScorer)
+      gameScorer.calculateSecondBowlScore(game, frameScorer)
+      game.updateStrikeAndSpareVariables(frameScorer)
+      frameScorer.inputBowls(7,2);
+      game.updateStrike(frameScorer)
+      gameScorer.calculateFirstBowlScore(game, frameScorer)
+      gameScorer.calculateSecondBowlScore(game, frameScorer)
+      game.updateStrikeAndSpareVariables(frameScorer)
+      frameScorer.inputBowls(10,0);
+      game.updateStrike(frameScorer)
+      gameScorer.calculateFirstBowlScore(game, frameScorer)
+      gameScorer.calculateSecondBowlScore(game, frameScorer)
+      game.updateStrikeAndSpareVariables(frameScorer)
+      frameScorer.inputBowls(10,0);
+      game.updateStrike(frameScorer)
+      gameScorer.calculateFirstBowlScore(game, frameScorer)
+      gameScorer.calculateSecondBowlScore(game, frameScorer)
+      game.updateStrikeAndSpareVariables(frameScorer)
+      frameScorer.inputBowls(10,0);
+      game.updateStrike(frameScorer)
+      gameScorer.calculateFirstBowlScore(game, frameScorer)
+      gameScorer.calculateSecondBowlScore(game, frameScorer)
+      game.updateStrikeAndSpareVariables(frameScorer)
+      frameScorer.inputBowls(9,0);
+      game.updateStrike(frameScorer)
+      gameScorer.calculateFirstBowlScore(game, frameScorer)
+      gameScorer.calculateSecondBowlScore(game, frameScorer)
+      game.updateStrikeAndSpareVariables(frameScorer)
+      frameScorer.inputBowls(8,2);
+      gameScorer.calculateFirstBowlScore(game, frameScorer)
+      gameScorer.calculateSecondBowlScore(game, frameScorer)
+      game.updateStrikeAndSpareVariables(frameScorer)
+      frameScorer.inputBowls(9,1);
+      game.updateStrike(frameScorer)
+      gameScorer.calculateFirstBowlScore(game, frameScorer)
+      gameScorer.calculateSecondBowlScore(game, frameScorer)
+      game.updateStrikeAndSpareVariables(frameScorer)
+      frameScorer.inputBowls(10,0);
+      game.updateStrike(frameScorer)
+      gameScorer.calculateFirstBowlScore(game, frameScorer)
+      gameScorer.calculateSecondBowlScore(game, frameScorer)
+      game.updateStrikeAndSpareVariables(frameScorer)
+      expect(gameScorer.total).toEqual(187)
+    });
+
+    it("test 2", function() {
+      frameScorer.inputBowls(1,4);
```

Comment:
You could either use helper functions or for loops for this type of test.
maybe a function that calls:
```
gameScorer.calculateFirstBowlScore(game, frameScorer)
gameScorer.calculateSecondBowlScore(game, frameScorer)
game.updateStrikeAndSpareVariables(frameScorer)
```

These could also be grouped at the game class level (even better) under a name like `gameScorer.calculateRound(game, frameScorer)`